recipes/ndf/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "ndf" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a87c0bd2cccbc438d24859cbe64de34157b432f104d0e7b59b0610264b538522
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py2k]
+
+requirements:
+  host:
+    - python
```

Noarch packages can use this syntax to limit the Python version:
```suggestion
    - python <3
```